.ci/validation.sh

```diff
@@ -42,7 +42,7 @@ check-missing-pitests)
 
   list=($(cat pom.xml | \
     xmlstarlet sel --ps -N pom="http://maven.apache.org/POM/4.0.0" \
-    -t -v '//pom:profile[./pom:id[contains(text(),'pitest')]]//pom:targetClasses/pom:param'))
+    -t -v '//pom:profile[./pom:id[contains(text(),'"'pitest'"')]]//pom:targetClasses/pom:param'))
```

Comment:
```suggestion
    -t -v "//pom:profile[./pom:id[contains(text(),'pitest')]]//pom:targetClasses/pom:param"))
```

Please send temporary commit to show that it works by removing some pitest target class.

Reply:
@shubh220922 ping